dali/kernels/slice/slice_cpu.h

```diff
@@ -347,6 +368,15 @@ void SliceKernel(SequentialExecutionEngine &exec_engine,
                  const SliceArgs<OutputType, Dims> &args,
                  int /* min_blk_sz */ = -1, int /* req_nblocks */ = -1) {
   (void) exec_engine;
+
+  // If the output and input data type is the same and the slice arguments take the whole extent
+  // of the input, then we can simply run memcpy.
+  if (CanRunPlainCopy<OutputType, InputType, Dims>(out_strides, in_strides, out_shape, in_shape,
+                                                   args)) {
```

Comment:
Nitpick: this line break looks weird... maybe moving all arguments to a new line with less indent or splitting between strides and shapes would look better?
```suggestion
  if (CanRunPlainCopy<OutputType, InputType, Dims>(out_strides, in_strides,
                                                   out_shape, in_shape,
                                                   args)) {
```
or
```suggestion
  if (CanRunPlainCopy<OutputType, InputType, Dims>(
      out_strides, in_strides, out_shape, in_shape, args)) {
```
?